examples/plugins/src/package-json/src/integration/dependencies.audit.ts

```diff
@@ -0,0 +1,143 @@
+import { Audit, AuditOutput, Issue } from '@code-pushup/models';
+import {
+  factorOf,
+  findLineNumberInText,
+} from '../../../../../../dist/packages/utils';
```

Comment:
It's because out code push up run fails in CI
[`Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@code-pushup/utils' imported from /home/runner/work/cli/cli/code-pushup.config.bundled_ux1s2l1z3u.mjs`](https://github.com/code-pushup/cli/actions/runs/7232302305/job/19706353010)

Reply:
Ah, I see. I think that should be fixed by adding your plugins to the build step: https://github.com/code-pushup/cli/blob/main/.github/workflows/ci.yml#L183